edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-﻿ARG base_tag=3.1.18-alpine3.13
+﻿ARG base_tag=3.1-alpine3.14
```

Comment:
Should we also update it for other Images ? EdgeHub, Proxy etc?

Reply:
We would need to do this later. We want to get this working for one component, one architecture (one base image).

Once we have something working, we make widespread changes.

Reply:
If you want to test for a single component, I'd suggest create a draft PR and test it out there.
By merging one component in the prod can create confusion and possible maintenance overhead.